File: CheckersPythonEngine/game_engine/TourDeJeu.py
from game_engine import Deplacement


class TourDeJeu:
    deplacements = []
    codeError: int
    success: bool

    def __init__(self):
        self.codeError = -1
        self.success = False

    def add_deplacement(self, d: Deplacement):
        self.deplacements.append(d)
        self.success = True
        return True

    def set_code_error(self, code: int):
        self.codeError = code

File: Checkers/Assets/Scripts/Game/TourDeJeu.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class TourDeJeu
{
    private List<Deplacement> deplacements = new List<Deplacement>();
    private int CodeError { get; set; }
    public bool Success { get; set; }

    public TourDeJeu()
    {
        CodeError = -1;
        Success = false;
    }

    public bool AddDeplacement(Deplacement d)
    {
        deplacements.Add(d);
        Success = true;
        return true;
    }

    public void SetCodeError(int code)
    {
        CodeError = code;
    }
}
